### PR TITLE
Minor typo and grammar fixes

### PR DESCRIPTION
**Pull request by idvoretskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Well, you need aria2. And a web browser (if that even counts!)
 # Docker support
 
 There is two Dockerfile in this project, one is a common Dockerfile, which can be use for **testing purpose**.<br>
-The second is a **production ready** Dockerfile for arm32v7 plateforms (which includes raspberry).
+The second is a **production ready** Dockerfile for arm32v7 platforms (which include raspberry pi).
```
Comment (no1xsyzy) [inline README.md]: I think `(including ...)` will be better.

**Comment by idvoretskyi** [comment]
@no1xsyzy thanks, updated.

**Comment by idvoretskyi** [comment]
@no1xsyzy @ziahamza a friendly ping :)

**Comment by no1xsyzy** [comment]
sorry but I don't have write access.
@anthonyraymond

**Pull request merged**